src/components/EntryEditor/EntryEditor.js

```diff
@@ -53,9 +53,15 @@ class EntryEditor extends Component {
     const controlClassName = `${ styles.controlPane } ${ this.state.showEventBlocker && styles.blocker }`;
     const previewClassName = `${ styles.previewPane } ${ this.state.showEventBlocker && styles.blocker }`;
 
+    const disablePreview = collection.getIn(['editor', 'disable_preview'], false);
+
+    const togglePreviewButton = (
+      <Button className={styles.previewToggle} onClick={this.handleTogglePreview}>Toggle Preview</Button>
+    );
+
     const editor = (
       <div className={controlClassName}>
-        <Button className={styles.previewToggle} onClick={this.handleTogglePreview}>Toggle Preview</Button>
+        { !disablePreview ? togglePreviewButton : null }
```

Comment:
This involves too many repeated negations for my taste - "not disable preview" is doubly negated and unnecessarily hard to follow.